.github/workflows/yjit-ubuntu.yml

```diff
@@ -1,105 +0,0 @@
-name: YJIT Ubuntu
-on:
-  push:
-    paths-ignore:
-      - 'doc/**'
-      - '**.md'
-      - '**.rdoc'
-  pull_request:
-    paths-ignore:
-      - 'doc/**'
-      - '**.md'
-      - '**.rdoc'
-
```

Comment:
@nobu Did you just remove the YJIT tests? If so, why?

@XrXr

Reply:
This is testing only MinGW builds.
Not only YJIT tests, but all other than mingw 😁